starter/slack-spring-boot-autoconfigure/src/main/kotlin/io/olaph/slack/broker/autoconfiguration/EvaluationReport.kt

```diff
@@ -0,0 +1,65 @@
+package io.olaph.slack.broker.autoconfiguration
+
+import io.olaph.slack.broker.receiver.EventReceiver
+import io.olaph.slack.broker.receiver.InstallationReceiver
+import io.olaph.slack.broker.receiver.InteractiveComponentReceiver
+import io.olaph.slack.broker.receiver.MismatchCommandReciever
+import io.olaph.slack.broker.receiver.SL4JLoggingReceiver
+import io.olaph.slack.broker.receiver.SlashCommandReceiver
+import org.springframework.context.ApplicationContext
+import org.springframework.context.ApplicationListener
+import org.springframework.context.event.ContextRefreshedEvent
+
+
+class EvaluationReport : ApplicationListener<ContextRefreshedEvent> {
```

Comment:
i think we also need to implement 
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/core/Ordered.html
with Lowest Precedence to ensure the execution is definitely after the beans are registered